extensions_containers.tf

```diff
@@ -83,7 +81,7 @@ resource "azurerm_role_assignment" "va_auto_provisioning_containers_role" {
   for_each = contains(var.mdc_plans_list, "Containers") ? local.container_roles : {}
 
   principal_id       = azurerm_subscription_policy_assignment.container[each.value.policy].identity[0].principal_id
-  scope              = data.azurerm_subscription.cur.id
+  scope              = data.azurerm_subscription.current.id
   role_definition_id = data.azurerm_role_definition.container_roles[each.key].id
 
```

Comment:
Is [line 88](https://github.com/oribenartzyM/terraform-azure-mdc-defender-plans-azure/blob/oribenazrtzy/add-roles-to-module/extensions_containers.tf#L87-L89) correct?:

```hcl
azurerm_security_center_subscription_pricing.asc_plans["VirtualMachines"]
```